### PR TITLE
remove button toggle and add stop button

### PR DESCRIPTION
**Pull request by kavithacm**
Signed-off-by: Kavitha Conjeevaram Mohan <mohakavi@amazon.com>

### Description
Replace EUI Button toggle with EUI button and add new stop button

### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by mengweieric — COMMENTED** [review]
Question: do we need to update cypress tests/jest tests, and update snapshots for these changes?

**Comment by kavithacm** [comment]
> Question: do we need to update cypress tests/jest tests, and update snapshots for these changes?

oh yes, a small change is required in cypress. i'll update it

File: dashboards-observability/.cypress/integration/event_analytics.spec.js [hunk 1]
```diff
@@ -400,7 +400,6 @@ describe('Switch on and off livetail', () => {
     cy.wait(delay * 2);
     cy.get('.euiToastHeader__title').contains('On').should('exist');
 
-    cy.get('[data-test-subj=eventLiveTail]').click();
```
Comment (ps48) [inline dashboards-observability/.cypress/integration/event_analytics.spec.js:403]: Why was this removed from cypress? Don't we switch on the live tail for cypress?

File: dashboards-observability/public/components/explorer/explorer.tsx [hunk 2]
```diff
@@ -1110,28 +1102,21 @@ export const Explorer = ({
     }
   };
 
+  const stopLive = () => {
+    setLiveTailName('Live');
+    setIsLiveTailOn(false);
+    setLiveHits(0);
+    setIsLiveTailPopoverOpen(false);
+    if (isLiveTailOnRef.current) setToast('Live tail Off', 'danger');
```
Comment (ps48) [inline dashboards-observability/public/components/explorer/explorer.tsx:1110]: Question: does this needs to be a red colored toast. Would this give an erroneous impression to users?

**Review by ps48 — APPROVED** [review]
Minor edits.

File: dashboards-observability/public/components/explorer/explorer.tsx (hunk 2 above)
Comment (kavithacm) [reply dashboards-observability/public/components/explorer/explorer.tsx:1110]: this was a change proposed by David to change it to red so that it would be different from the On toast message
